code/style.css

```diff
@@ -1,5 +1,124 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@import url('https://fonts.googleapis.com/css2?family=Parisienne&display=swap');
 
 body {
-  background-color: lightblue;
+  background-color: rgba(243, 234, 204, 0.897);
+  font-family: "Parisienne", cursive;
+  color: black;
 }
+
+menu {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+menu a {
+  display: inline-block;
+  padding: 10px 15px;
+  background: rgba(243, 234, 204, 0.897);
+  color: black;
+  font-weight: bold;
+  text-decoration: none;
+  border-radius: 3px;
+  margin: 0 5px;
+}
+
+menu a:hover {
+  background:black;
+  color:rgba(243, 234, 204, 0.897);
+}
+
+@keyframes scale {
+  10% {transform: scale(2);} 
+  100% {transform: scale(1);}
+}
```

Comment:
Cool with an animation! The scaling makes the image cover the other elements so maybe you want to scale it a little bit less, maybe 1.2?
```
@keyframes scale {
  10% {transform: scale(1.2);} 
  100% {transform: scale(1);}
}
```